### PR TITLE
Sofia branch

### PR DESCRIPTION
**Pull request by SofiaPSU**
I forgot to make several pull requests but made several commits.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -6,14 +6,25 @@
...
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
+      <h1>Sofia Riley</h1>
+      <p style = "font-size: 30px;">Important Links:</p>
+      <a href ="https://www.linkedin.com/in/sofiariley/"> LinkedIn</a>
+      <a href ="https://github.com/SofiaPSU/sps-portfolio"> Github</a>
+      <p style ="font-size:30px">Introduction</p>
+      
+      <p>Hi Everyone! My name is Sofia and I am sophomore at Penn State!</p>
+      <p> I am currently majoring in Computer Science and planning to minor in Cyber Security.</p>
```
Comment (GoogleyGabriel) [inline portfolio/src/main/webapp/index.html:19]: Remove unnecessary spaces between tags `<p>.`

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -6,14 +6,25 @@
...
+      <p> On campus I am involved with the Assocation of Women in Computing, the Penn State Equestrian Team, the Girls Who Code College Loop, and the Women in Engineering Program.</p>
+      <p> I am a Technology Lead for the Girls Who Code College Loop and a Facilitator for the study groups organized by the Women in Engineering Program.</p>
+      
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      <img style ='position:absolute; top:25px; left: 25px; width:300px; height:250px' src= "images/profile.jpg">
+
```
Comment (GoogleyGabriel) [inline portfolio/src/main/webapp/index.html:27]: Try to not add extra blank lines unless it helps with visibility.

**Review by GoogleyGabriel — APPROVED** [review]
> I forgot to make several pull requests but made several commits.

You actually did exactly what I asked for! This is exactly how it should be done, good job Sofia! You want to commit as often as possible, and have small PRs (you did both).

Only extra thing I would say is make sure that to give a title and a meaningful description to your PR.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -6,14 +6,25 @@
...
+      <h1>Sofia Riley</h1>
+      <p style = "font-size: 30px;">Important Links:</p>
+      <a href ="https://www.linkedin.com/in/sofiariley/"> LinkedIn</a>
+      <a href ="https://github.com/SofiaPSU/sps-portfolio"> Github</a>
+      <p style ="font-size:30px">Introduction</p>
+      
+      <p>Hi Everyone! My name is Sofia and I am sophomore at Penn State!</p>
+      <p> I am currently majoring in Computer Science and planning to minor in Cyber Security.</p>
+      <p> On campus I am involved with the Assocation of Women in Computing, the Penn State Equestrian Team, the Girls Who Code College Loop, and the Women in Engineering Program.</p>
+      <p> I am a Technology Lead for the Girls Who Code College Loop and a Facilitator for the study groups organized by the Women in Engineering Program.</p>
```
Comment (sebagabs) [inline portfolio/src/main/webapp/index.html:21]: @GoogleyGabriel mentioned me that lines should be less than 100 characters, so keep that in mind; it appened to me too!

**Review by sebagabs — APPROVED** [review]
I think Gabriel covered the details, the rest is looking good!

**Pull request merged**